File: managerapp-server/src/main/java/com/managerapp/Model/Brand.java
package com.managerapp.Model;

import lombok.*;
import javax.persistence.*;
import java.io.Serializable;

@Getter
@Setter
@NoArgsConstructor
@EqualsAndHashCode
@AllArgsConstructor
@Entity
@Table(name = "brand", schema = "managerapp")
public class Brand implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO, generator = "managerapp.brand_brandid_seq")
    private Long brandId;
    private String brandName;
    private Boolean crueltyFree;

}

File: managerapp-frontend/src/app/model/brand.ts
export class Brand {
  brandId : number;
  brandName : String;
  crueltyFree? : boolean;
  constructor(brandId: number, brandName:string, crueltyFree: boolean){
    this.brandId = brandId;
    this.brandName = brandName;
    this.crueltyFree = crueltyFree;
  }
}
